### PR TITLE
Disable publishing of NuGet packages on Pull Requests

### LINKED ISSUE #858: Do not publish NuGet packages to AppVeyor feed for PRs
We just want NuGet packages made from master.

Supports #857.

### PR DESCRIPTION
**Pull request by blairconrad**
Fixes #858.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -9,6 +9,9 @@ install:
 build_script:
     - cmd: build.netstd.cmd
 
+nuget:
+    disable_publish_on_pr: true
```
Comment (blairconrad) [inline appveyor.yml]: Checks "Do not publish NuGet package artifacts to account/project feeds on Pull Requests" on https://ci.appveyor.com/project/FakeItEasy/fakeiteasy/settings/nuget

**Comment by blairconrad** [comment]
> LTGM

Oh, you French and your non-standard (from an English perspective) word order.

Thanks for the merge!

**Comment by thomaslevesque** [comment]
> Oh, you French and your non-standard (from an English perspective) word order.

I meant TMGL: "To me, good it looks" :wink:

**Comment by blairconrad** [comment]
Y'know, I had thought that that would've been better.

**Pull request merged**